### DISCUSSION TITLE
Differential Geometry

### DISCUSSION THREAD

**Discussion by ghost** [kaveh808/kons-9#253] [Ideas]
A general discussion of directly exploring differential geometry with kons-9 and Common Lisp.  Ultimately I would like to work through the book (which generously has a CC-style license, so we could even rewrite it for CL):

Functional Differential Geometry, Gerald Jay Sussman and Jack Wisdom with Will Far, 2013

https://mitp-content-server.mit.edu/books/content/sectbyfn/books_pres_0/9580/9580.pdf

using Kons-9 and Common Lisp.  This is probably a long term project.

[spoiler]  The design on the cover traces a lambda.

**Comment by kaveh808** [comment]
Sounds like an interesting addition to kons-9. And an ambitious one.

**Comment by theottm** [comment]
Very nice exercise!

**Comment by ghost** [comment]


**Reply by kaveh808** [reply]
Yes, it is a bummer that CCL does not run on Apple M silicon. I used various versions of it since the early 90's.

**Reply by kaveh808** [reply]
Go for it. :)
